Readme.md

```diff
@@ -87,90 +119,38 @@ and include this directory to be first in the system PATH in the shell where you
 build webrtc, before invoking `build-webrtc.sh`:  
 `export PATH=/path/to/dir-with-python-symlink:$PATH`  
 
-### Third party deps ###
-On any platform other than Linux, set the following two env vars, to make the webrtc build system
-find libs not located in standard system paths:  
-`export WEBRTC_DEPS_INCLUDE=/path/to/prefix/include`
-`export WEBRTC_DEPS_LIB=/path/to/prefix/lib`  
-Depending on the plaform, `/path/to/prefix` will be:  
-- iOS, Android, Windows: `$builddir/usr`, where `$builddir` is the builddir parameter that you passed to the `setup-deps.sh` script.
-- MacOS: The prefix where MacPorts/HomeBrew installs libraries.  
+## Manual build the MEGAchat codebase, including a test app ##
 
-*Android*  
-You need to set ANDROID_NDK to an NDK installation. The current webrtc revision
-has been tested with NDK r10d.
+In case you want to compile example QTapp manually, you can follow the steps below.
 
-### Verify the webrtc build ###
-* Cd to the directory you passed to build-webrtc.sh, and then:  
-   - non-iOS  
-  `cd src/out/Release|Debug`  
-  - iOS  
-  `cd src/out/Release-iphoneos|Debug-iphoneos`  
+Change directory to the root of the MEGAchat checkout  
 
-* Built executables  
+ - `mkdir ./build`    
+ - `cd ./build`    
+ - `qmake ./contrib/QtCreator/MEGAchat.pro -spec linux-g++ CONFIG+=qml_debug CONFIG+=force_debug_info CONFIG+=separate_debug_info && /usr/bin/make _all`    
```

Comment:
`./contrib` won't exist in the "cd'ed" build.   `..`?

Reply:
You're right. Updating!

Reply:
You're right. Updating